java/org/contikios/cooja/Simulation.java

```diff
@@ -124,6 +130,8 @@ public void execute(long t) {
         try {
           Thread.sleep(sleep);
         } catch (InterruptedException e) {
+          // Restore interrupted status
+          Thread.currentThread().interrupt();
```

Comment:
I think these two linens can go in a separate PR that is merged to master. (But shouldn't there be a return in the catch clause to prevent more events from being scheduled?)

Reply:
I made a separate PR. I think the event should be rescheduled in case the caller will continue running the simulation regardless of the interruption.